src/modules/Dropdown/Dropdown.js

```diff
@@ -849,9 +863,11 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
-        onMouseDown={e => e.preventDefault()} // prevent default to allow item select without closing on blur
+        // prevent default to allow item select without closing on blur
+        onMouseDown={e => e.preventDefault()}
```

Comment:
I would have thought so, but it didn't seem to be the case.  I'll look at it again and see.